src/main/java/seedu/address/logic/parser/SetAttCommandParser.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.*;
```

Comment:
I think you should avoid using * for import statement in order to follow the code standards